.githooks/pre-commit

```diff
@@ -0,0 +1,12 @@
+#!/Users/jameskatarikawe/anaconda2/bin/python
```

Comment:
I'm not sure this will work on other people's machines. Try reading through this `https://stackoverflow.com/questions/2429511/why-do-people-write-the-usr-bin-env-python-shebang-on-the-first-line-of-a-pyt` or find a better alternative if possible

Reply:
thanks. This helped